server/fileCompress/app.py

```diff
@@ -0,0 +1,103 @@
+import os
+import zipfile
+import json
+import posixpath
+from pprint import pprint
```

Comment:
I'm pretty sure posixpath and pprint are unsued.

Reply:
You are correct, I had them for a hello world application